templates/entry.ttl

```diff
@@ -2,6 +2,12 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 :{{ ID }}
     skos:prefLabel "{{ PREF_LABEL }}"@en;
+#    skos:altLabel "{{ ALT_LABEL }}";
```

Comment:
Can you make this into a second template and rename them something like:
+ `basic.ttl`
+ `full.ttl`

My current feeling is that most of SKOS / ttl is too complicated. Just (name, definition) pairs is > 90% of the value.
I'm happy to give it go, but think we should keep a SKOS/TTL light mode.  

^^ thats my wooly justification of the change request

Reply:
I thinks thats a fair point. There will be many entries that do not require more complex relational modeling